src/main/java/org/github/guppy/backend/controller/ProductController.java

```diff
@@ -0,0 +1,14 @@
+package org.github.guppy.backend.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class ProductController {
+
+    @GetMapping("/v1/products")
+    public ResponseEntity<String> products() {
```

Comment:
A good idea could be rename this method to `getProducts()` 😄

Reply:
Ok! It's done